storage/localstore/mode_get.go

```diff
@@ -96,6 +97,15 @@ func (db *DB) get(mode chunk.ModeGet, addr chunk.Address) (out shed.Item, err er
 			}
 		}()
 
+	case chunk.ModeGetPin:
+		pinChunk, err := db.pinIndex.Get(item)
```

Comment:
pinChunk is the Index type. Name is misleading.

Reply:
changed to pinnedChunk

Reply:
I made a mistake in the comment, I was thinking of Item type, so pinItem instead pinChunk, or pinnedItem. Not of so high importance.

Reply:
aaah.. ok ok . It's an item.. not a chunk.
Done